examples/ScreenText.js

```diff
@@ -33,6 +33,7 @@ requirejs(['./WorldWindShim',
             // Imagery layers.
             {layer: new WorldWind.BMNGLayer(), enabled: true},
             {layer: new WorldWind.BMNGLandsatLayer(), enabled: false},
+            {layer: new WorldWind.AtmosphereLayer(), enabled: true},
             // WorldWindow UI layers.
             {layer: new WorldWind.BingAerialWithLabelsLayer(null), enabled: true},
```

Comment:
The BingAerialWithLabelsLayer is mistakenly added after the AtmosphereLayer, which produces the latter to not show its inner shading. BingAerialWitlLabelsLayer should go before the atmosphere and after BMNGLandsatLayer.